### ISSUE TITLE
Type error when using redis queues.

### ISSUE THREAD

**Issue by TraffordFewster** [putyourlightson/craft-blitz#522]
When saving an entry the queue job is dispatched to refresh the cache, but looking in my queue.log file there appears to be an error from Blitz and the cache doesn't seem to always clear.

### Error
```
2023-06-23 08:49:40 [queue.ERROR] [TypeError] TypeError: putyourlightson\blitz\jobs\RefreshCacheJob::_populateCacheIdsFromElementQueryCaches(): Argument #2 ($queue) must be of type craft\queue\QueueInterface, yii\queue\redis\Queue given, called in site/vendor/putyourlightson/craft-blitz/src/jobs/RefreshCacheJob.php on line 74 and defined in site/vendor/putyourlightson/craft-blitz/src/jobs/RefreshCacheJob.php:146
Stack trace:
#0 site/vendor/putyourlightson/craft-blitz/src/jobs/RefreshCacheJob.php(74): putyourlightson\blitz\jobs\RefreshCacheJob->_populateCacheIdsFromElementQueryCaches(Object(putyourlightson\blitz\models\RefreshDataModel), Object(yii\queue\redis\Queue))
#1 site/vendor/yiisoft/yii2-queue/src/Queue.php(246): putyourlightson\blitz\jobs\RefreshCacheJob->execute(Object(yii\queue\redis\Queue))
#2 site/vendor/yiisoft/yii2-queue/src/cli/Queue.php(162): yii\queue\Queue->handleMessage('33', 'O:42:"putyourli...', '300', '1')
#3 site/vendor/yiisoft/yii2-queue/src/cli/Command.php(146): yii\queue\cli\Queue->execute('33', 'O:42:"putyourli...', '300', '1', '77846')
#4 [internal function]: yii\queue\cli\Command->actionExec('33', '300', '1', '77846')
#5 site/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#6 site/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#7 site/vendor/yiisoft/yii2/console/Controller.php(180): yii\base\Controller->runAction('exec', Array)
#8 site/vendor/yiisoft/yii2/base/Module.php(552): yii\console\Controller->runAction('exec', Array)
#9 site/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('queue/exec', Array)
#10 site/vendor/craftcms/cms/src/console/Application.php(90): yii\console\Application->runAction('queue/exec', Array)
#11 site/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('queue/exec', Array)
#12 site/vendor/craftcms/cms/src/console/Application.php(121): yii\console\Application->handleRequest(Object(craft\console\Request))
#13 site/vendor/yiisoft/yii2/base/Application.php(384): craft\console\Application->handleRequest(Object(craft\console\Request))
#14 site/craft(13): yii\base\Application->run()
#15 {main} {"memory":17823272,"exception":"[object] (TypeError(code: 0): putyourlightson\\blitz\\jobs\\RefreshCacheJob::_populateCacheIdsFromElementQueryCaches(): Argument #2 ($queue) must be of type craft\\queue\\QueueInterface, yii\\queue\\redis\\Queue given, called in site/vendor/putyourlightson/craft-blitz/src/jobs/RefreshCacheJob.php on line 74 at site/vendor/putyourlightson/craft-blitz/src/jobs/RefreshCacheJob.php:146)"} 
```

### `app.php` config:
```
[
    'id' => App::env('CRAFT_APP_ID') ?: 'CraftCMS',
    'components' => [
        'redis' => [
            'class' => yii\redis\Connection::class,
            'hostname' => 'localhost',
            'port' => 6379,
            'database' => 1,
        ],
        'queue' => [
            'class' => yii\queue\redis\Queue::class,
            'redis' => 'redis', 
            'channel' => 'my_queue', 
        ],
    ],
];
```

### Versions
- `craftcms/cms` 4.4.14
- `putyourlightson/craft-blitz` 4.4.5
- `yiisoft/yii2-redis` 2.0.18
- PHP 8.1.19
- MySQL 5.7.23
- Redis 7.0.0

**Comment by bencroker** [comment]
Thanks for reporting this. It turned out to be an issue with the Redis queue driver. Fixed in https://github.com/putyourlightson/craft-blitz/commit/4401a872da7585c526f4138346679597375ae9ca for the next release.

**Comment by bencroker** [comment]
Released in version [4.4.7](https://github.com/putyourlightson/craft-blitz/releases/tag/4.4.7).
